tensorflow/c/c_test_util.cc

```diff
@@ -36,17 +36,31 @@ TF_Tensor* Int8Tensor(const int64_t* dims, int num_dims, const char* values) {
   return t;
 }
 
-TF_Tensor* Int32Tensor(int32_t v) {
-  const int num_bytes = sizeof(int32_t);
-  int32_t* values = new int32_t[1];
-  values[0] = v;
-  return TF_NewTensor(TF_INT32, nullptr, 0, values, num_bytes,
-                      &Int32Deallocator, nullptr);
+TF_Tensor* Int32Tensor(int32_t v, const tensorflow::TensorShape shape) {
+  int num_dims = shape.dims();
+  int64_t* dims = new int64_t[num_dims];
```

Comment:
I think you can use shape.dim_sizes() directly instead of creating your own array (here and below).